crown-court-contribution/src/main/java/uk/gov/justice/laa/crime/contribution/service/ContributionService.java

```diff
@@ -82,4 +84,27 @@ public boolean checkReassessment(final int repId, final String laaTransactionId)
         return false;
     }
 
+    public boolean cds15WorkAround(RepOrderDTO repOrderDTO) {
```

Comment:
The method name can be renamed to `isCds15WorkAround` or something similar as it returns a boolean

Reply:
Done